lib/cucumber/runtime.rb

```diff
@@ -229,6 +229,7 @@ def filters
         filters << Cucumber::Core::Test::LocationsFilter.new(filespecs.locations)
         filters << Filters::Randomizer.new(@configuration.seed) if @configuration.randomize?
         filters << Filters::Quit.new
+        filters << Filters::Retry.new(@configuration)
```

Comment:
I expect I'm missing something, but won't the Quit filter be invoked as soon as the first attempt to run the test case is passed on by the retry filter?